kata_agent.go

```diff
@@ -445,6 +445,22 @@ func (k *kataAgent) createContainer(pod *Pod, c *Container) error {
 	// We need to give the OCI spec our absolute rootfs path in the guest.
 	grpcSpec.Root.Path = rootPath
 
+	// Remove Hooks
+	grpcSpec.Hooks = nil
+
+	// Remove Seccomp
+	grpcSpec.Linux.Seccomp = nil
```

Comment:
Any reason why we're removing SecComp? It should not be an issue to apply those inside the VM.

Reply:
Unfortunately, we need several things to make the seccomp usable (as explained here https://github.com/kata-containers/agent/issues/104).
This is temporary until we can support seccomp with Kata Containers agent.